packages/fxa-settings/src/pages/ResetPassword/AccountRecoveryResetPassword/en.ftl

```diff
@@ -5,3 +5,9 @@ create-new-password-header = Create new password
 account-restored-success-message = You have successfully restored your account using your account recovery key. Create a new password to secure your data, and store it in a safe location.
 # Feedback displayed in alert bar when password reset is successful
 account-recovery-reset-password-success-alert = Password set
+# An error case was hit that we cannot account for.
+account-recovery-reset-password-unexpected-error = Unexpected Error Encountered
+account-recovery-reset-password-email-resent = Email resent. Add accounts@firefox.com to your contacts to ensure a smooth delivery.
+account-recovery-reset-password-email-resend-error = Sorry, there was a problem resending a reset password link to your email.
+account-recovery-reset-password-invalid-context = Invalid context: { $keys }
```

Comment:
What does `context` mean here? This is a highly technical meaning, so it's going to be hard to translate this accurately. Could you provide a comment on what `invalid context` is referring to, and also an explanation of what will appear after in `{ $keys }`?

Reply:
This is a technical message, and in reality should not ever occur in production. I am actually thinkng of approaching this in a more developer centric way.

Reply:
This string has been removed in favor the generic `Unexpected error encountered` message.